opus/application/static_media/js/browse.js

```diff
@@ -624,12 +626,26 @@ var o_browse = {
 
     // find the first displayed observation index & id in the upper left corner
     updateSliderHandle: function() {
+        let tab = `#${opus.prefs.view}`;
+        let contentsView = o_browse.getScrollContainerClass();
         let selector = (opus.prefs.browse === "dataTable") ? `#${opus.prefs.view} #dataTable tbody tr` : `#${opus.prefs.view} .gallery .thumbnail-container`;
+
+        // Fot gallery view, the topBoxBoundary is the top of .gallery-contents
+        // For table view, we will set the topBoxBoundary to be the bottom of thead (account for height of thead)
+        let topBoxBoundary = (opus.prefs.browse === "dataTable") ? $(".gallery-contents").offset().top + $(`${tab} #dataTable thead th`).outerHeight() : $(".gallery-contents").offset().top;
```

Comment:
is the top of the box not the same as the PS container or the infinite scroll container?  also note that this code is not tab-specific.

Reply:
The box (.gallery-contents) are the same for both gallery and table. But in table view, there is that one row of sticky table header which will always cover one item of the table as we scroll down, so we have to set the top boundary to be the bottom of the table header. That way we can always get the top item of the table seen by the user.